pages/rating.vue

```
@@ -140,7 +141,7 @@ export default {
 
     async sendForm() {
       const validation = await this.validateForm();
-      if (!validation) {
+      if (!validation || this.submitted) {
         return;
```

It will be silenced error, is it possible to do something like disabling the sending button ? Write a $toast.error ?